modules/Layout/Boilerplate/Boilerplate.tsx

```diff
@@ -58,7 +58,7 @@ function Boilerplate() {
                             )}
                         </div>
                     )}
-                    {hasContactInformation && (
+                    {(hasContactInformation || companyInformation.website) && (
```

Comment:
I was wondering if it's better to update `hasAnyContactInformation` from `theme-kit-nextjs`, but there is a question if Website should be part of the contact, and if changing it wouldn't break something else.

Reply:
I think it's okay to update the helper in `theme-kit`. This block is pretty much the same across all themes, so should be safe to update.

Reply:
Also, it would be nice to move the `companyInformation.website` check above together with other checks (e.g. `hasEmail`) to keep things consistent